generators/entity-server/templates/dotnetcore/src/Project/Controllers/EntityController.cs.ejs

```diff
@@ -93,7 +93,11 @@ namespace <%= namespace %>.Controllers {
             <%_ if (hasDto) { _%>
             <%= pascalizedEntityClass %> <%= camelCasedEntityClass %> = _mapper.Map<<%= pascalizedEntityClass %>>(<%= asDto(camelCasedEntityClass) %>);
             <%_ } _%>
-            <%_ if (relationships.length != 0) { _%>
+            <%_ if (relationships.length != 0) { 
+                relationships.forEach( relationship => {
+                    if (relationship.otherEntityNamePascalized === 'User' && relationship.relationshipType !== 'many-to-many') { _%>
+            <%= camelCasedEntityClass %>.User = null;
```

Comment:
Maybe check if user is admin, if not, set the user to the current http request user.

Reply:
> Maybe check if user is admin, if not, set the relationship user to the current http request user?
> Maybe throw security-related exception if user is different (and show error message at the client)?

Hello
check all of item but same error exist.